src/olympia/reviewers/views.py

```diff
@@ -173,6 +174,12 @@ def dashboard(request):
             'https://wiki.mozilla.org/Add-ons/Reviewers/Guide'
         ),
         ))
+        sections[ugettext('Need Human Review')] = [(
+            ugettext('Need Human Review ({0})').format(
```

Comment:
I'm still not convinced about "Needing Human Review" as a thing - *all* of the queues contain addons that need some sort of human review.

Reply:
That's fair. I haven't found a good alternative yet though :(

Reply:
"Blocked on Review", or "Signature Pending Review"?

Reply:
The thing is, it doesn't necessarily block approval. Delaying or outright blocking automated approval is yet another step in the actions scanner rules can trigger. This is more of a "the scanner found something, but can't be 100% sure, need a human to double-check". The version can be auto-approved despite this.

Reply:
Ah, I see. Other suggestions: "Flagged for Human Review", "Flagged by Scanners", "Auto-flagged". All that said, the wording doesn't matter that much since reviewers will get used to whatever we pick very quickly.

Reply:
"Scanner Flagged" (similar naming to Admin Flagged)
"Scanner Review" or "Malware Review" (similar naming to Content Review)